doc/manual/using_subsystems.rst

```diff
@@ -0,0 +1,194 @@
+Datasets and subsystems
+=======================
+
+Beyond running single experiments, or basic use of master, client, and dashboard, ARTIQ supports a much broader range of interactions between its different components. These integrations allow for system control which is extensive, flexible, and dynamic, with space for different workflows and methodologies depending on the needs of particular sets of experiments. We will now explore some of these further tools and systems.
+
+.. note::
+
+    This page follows up directly on :doc:`getting_started_mgmt`, but discusses a broader range of features, most of which continue to rest on the foundation of the management system. Some sections (datasets, the browser) are still possible to try out using your PC alone; others (MonInj, the RTIO analyzer) are only meaningful in relation to some kind of real-time hardware.
+
+.. _mgmt-datasets:
+
+Datasets and results
+--------------------
+
+ARTIQ uses the concept of *datasets* to manage the data exchanged with experiments, both supplied *to* experiments (generally, from other experiments) and saved *from* experiments (i.e. results or records). We will now see how to view and manipulate datasets, both in experiments or through the management system. Create a new experiment as follows: ::
+
+    from artiq.experiment import *
+    import numpy as np
+    import time
+
+    class Datasets(EnvExperiment):
+        """Dataset tutorial"""
+        def build(self):
+            pass  # no devices used
+
+        def run(self):
+            self.set_dataset("parabola", np.full(10, np.nan), broadcast=True)
+            for i in range(10):
+                self.mutate_dataset("parabola", i, i*i)
+                time.sleep(0.5)
+
+Save it as ``dataset_tutorial.py``. Commit and push your changes, or rescan the repository, whichever is appropriate for your management system configuration. Submit the experiment. In the same window as 'Explorer', navigate to the 'Dataset' tab and observe that a new dataset has been created under the name ``parabola``. As the experiment runs, the values are progressively calculated and entered.
```

Comment:
A numbered list may be relevant given the sequential steps described here.

Reply:
It's not really a format used elsewhere in the tutorials, even though there's a lot of sequential instructions. It also implies a formulaicism which IMO isn't really present, these steps won't need to be repeated...